frame/support/src/traits/schedule.rs

```diff
@@ -426,6 +426,10 @@ pub mod v3 {
 		///
 		/// Will return an `Unavailable` error if the `address` is invalid.
 		fn next_dispatch_time(address: Self::Address) -> Result<BlockNumber, DispatchError>;
+
+		/// Return the maximum number of tasks possible to schedule per block.
+		#[cfg(feature = "runtime-benchmarks")]
+		fn max_scheduled_per_block() -> u32;
```

Comment:
I don't see a better solution, how to access to this configuration from the referenda benchmarks (and keep it seamless for the client). Any ideas?